recipe/meta.yaml

```diff
@@ -1,29 +1,29 @@
 {% set name = "clifford" %}
-{% set version = "1.0.5" %}
+{% set version = "1.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/pygae/{{ name }}/archive/v{{ version }}.zip
-  sha256: 3d9af40da1076c59bdb44697afa09d1426473098c93aa3ca2c157baa807941ad
+  sha256: 27251c2416d3c1927d5cfc783f655699b6a186fd59fcf733705959cb70924bf5
 
 build:
   noarch: python
   number: 1
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - python
```

Comment:
You'll need the `>=3.5` in the `host` sections as well.

Reply:
Why does this need to be specified in both?

Reply:
I'm not super familiar with the whole build process, so I'm not entirely sure. It may just constrain the build process to ensure the correct python version is used.